collector/remote_fx.go

```diff
@@ -0,0 +1,362 @@
+			[]string{"session_name"},
+			nil,
+		),
+		CurrentTCPRTT: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_current_tcp_rtt"),
+			"Average TCP round-trip time (RTT) detected in seconds.",
+			[]string{"session_name"},
+			nil,
+		),
+		CurrentUDPBandwidth: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_current_udp_bandwidth"),
+			"UDP Bandwidth detected in thousands of bits per second (1000 bps).",
+			[]string{"session_name"},
+			nil,
+		),
+		CurrentUDPRTT: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_current_udp_rtt"),
+			"Average UDP round-trip time (RTT) detected in seconds.",
+			[]string{"session_name"},
+			nil,
+		),
+		TotalReceivedBytes: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_total_received_bytes"),
+			"(TotalReceivedBytes)",
+			[]string{"session_name"},
+			nil,
+		),
+		TotalSentBytes: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_total_sent_bytes"),
+			"(TotalSentBytes)",
+			[]string{"session_name"},
+			nil,
+		),
+		UDPPacketsReceivedPersec: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_udp_packets_received_total"),
+			"Rate in packets per second at which packets are received over UDP.",
+			[]string{"session_name"},
+			nil,
+		),
+		UDPPacketsSentPersec: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_udp_packets_sent_total"),
+			"Rate in packets per second at which packets are sent over UDP.",
+			[]string{"session_name"},
+			nil,
+		),
+
+		//gfx
+		AverageEncodingTime: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "gfx_average_encoding_time"),
+			"Average frame encoding time in seconds",
+			[]string{"session_name"},
+			nil,
+		),
+		FrameQuality: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "gfx_frame_quality"),
+			"Quality of the output frame expressed as a percentage of the quality of the source frame.",
+			[]string{"session_name"},
+			nil,
+		),
+		FramesSkippedPerSecondInsufficientClientResources: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "gfx_frames_skipped_insufficient_clt_res_total"),
+			"Number of frames skipped per second due to insufficient client resources.",
+			[]string{"session_name"},
+			nil,
+		),
+		FramesSkippedPerSecondInsufficientNetworkResources: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "gfx_frames_skipped_insufficient_net_res_total"),
+			"Number of frames skipped per second due to insufficient network resources.",
+			[]string{"session_name"},
+			nil,
+		),
+		FramesSkippedPerSecondInsufficientServerResources: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "gfx_frames_skipped_insufficient_srv_res_total"),
+			"Number of frames skipped per second due to insufficient server resources.",
+			[]string{"session_name"},
+			nil,
+		),
```

Comment:
Would it make sense to merge these into a `gfx_frames_skipped_total` with a `reason` label? From an alerting perspective that might make it easier to write a query?